src/lib/components/particles/change-bar/style.module.scss

```diff
@@ -0,0 +1,10 @@
+.wrapper {
+	// width: 100%;
+	// background-color: grey;
+}
+
+.bar {
+	position: relative;
+	height: 1em;
```

Comment:
is there a world where we want this bar height to be customisable? - up to you

Reply:
Yes good point. That was a working value.